docs/emulator.md

```diff
@@ -70,3 +70,6 @@ The emulator also provides advanced options when start, for example, you can use
 7. In the browser, navigate to `http://localhost:7071/api/index` to play with the demo.
 
 ![image](https://user-images.githubusercontent.com/668244/93066361-78d11e80-f6ac-11ea-8b2e-bff28196850c.png)
+
+## Supporting persistent transport types
```

Comment:
The suggestion on persistent mode local debug is missing in the latest doc. How about combining this part with the first paragraph in the latest doc?
> When developing serverless applications, we provide an Azure SignalR Local Emulator to make the local development and integration easier. Please note that the emulator works only for serverless scenarios, for Default mode that the Azure SignalR Service acts as a proxy, you can directly use self-host SignalR to do local development. Please also note that emulator only works for Transient transport type (the default one) and doesn't work for Persistent transport type.

Reply:
Seems good to me.

Reply:
Sounds good to me.